packages/osmosis-std/src/types/cosmos/base/query/v1beta1.rs

```diff
@@ -80,12 +80,8 @@ pub struct PageResponse {
     /// next_key is the key to be passed to PageRequest.key to
     /// query the next page most efficiently. It will be empty if
     /// there are no more results.
-    #[prost(bytes = "vec", tag = "1")]
-    #[serde(
-        serialize_with = "crate::serde::as_base64_encoded_string::serialize",
-        deserialize_with = "crate::serde::as_base64_encoded_string::deserialize"
```

Comment:
I think these custom serialize/deserialize function shouldn't be missing? Maybe a new function that also handles `Option<Vec<u8>>`

Reply:
I changed the ordering of the transformers to resolve this